packages/domain/lib/src/models/people_model/ids.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'ids.g.dart';
+
+@JsonSerializable()
+class Ids {
+  Ids({
+    this.trakt,
+    this.slug,
+    this.imdb,
+    this.tmdb,
+    this.tvrage,
+  });
+
+  final int? trakt;
+  final String? slug;
+  final String? imdb;
+  final int? tmdb;
+  final dynamic tvrage;
```

Comment:
why dynamic?)

Reply:
just becsuse I don't know what type is it. I view json files and it was null anywhere

Reply:
will make some more research =)

Reply:
better to remove this field, I thought =)

Reply:
bad idea, you should find type of this field

Reply:
this is "int" =)